CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 #### Fixes
 
+* [#1430](https://github.com/ruby-grape/grape/pull/1430): Fix for declared(params) inside route_param. - [@Arkanain](https://github.com/Arkanain).
```

Comment:
Sorry to be a pain. There shouldn't be a period after `route_param`, also maybe quote the code as below?